docs/running_psalm/dealing_with_code_issues.md

```diff
@@ -114,26 +114,18 @@ If you want something more custom, like suppressing a certain type of error on c
 <?php
 namespace Foo\Bar;
 
-use PhpParser\Node;
-use Psalm\Codebase;
-use Psalm\FileSource;
 use Psalm\Plugin\Hook\AfterClassLikeVisitInterface;
-use Psalm\Storage\ClassLikeStorage;
+use Psalm\Plugin\Hook\TransfertObject\AfterClassLikeVisitEvent;
```

Comment:
It's 'transfer' in English

Reply:
Damn! Excuse my French 😄

Reply:
Having given this a bit more thought I think it would be even better if the namespace was `Event` or `Events`.

Reply:
Done